test/evergreen.yml

```diff
@@ -3430,7 +3429,183 @@ tasks:
+            ${test_env_vars|} test/cppsuite/run -t hs_cleanup -f test/cppsuite/configs/hs_cleanup_default.txt
+            ${test_env_vars|} ./test/csuite/wt1965_col_efficiency/test_wt1965_col_efficiency
+            ${test_env_vars|} ./test/csuite/wt2999_join_extractor/test_wt2999_join_extractor
+            ${test_env_vars|} ./test/csuite/wt3135_search_near_collator/test_wt3135_search_near_collator
+            ${test_env_vars|} ./test/csuite/wt3184_dup_index_collator/test_wt3184_dup_index_collator
+            ${test_env_vars|} ./test/csuite/wt4156_metadata_salvage/test_wt4156_metadata_salvage
+            ${test_env_vars|} ./test/csuite/wt2403_lsm_workload/test_wt2403_lsm_workload
+
+            ${test_env_vars|} WIREDTIGER_HOME=ex1 examples/c/ex_all/ex_all & ${test_env_vars|} WIREDTIGER_HOME=ex2 examples/c/ex_schema/ex_schema
+            # Rest of the examples
+            cd examples/c/
+            ${test_env_vars|} $CTEST --output-on-failure 2>&1
+            cd ../../
+
+            # Faster csuite tests. 30% in ~1 minute
+            ${test_env_vars|} $(pwd)/test/csuite/wt2403_lsm_workload/test_wt2403_lsm_workload 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt6185_modify_ts/test_wt6185_modify_ts 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt3874_pad_byte_collator/test_wt3874_pad_byte_collator 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt4105_large_doc_small_upd/test_wt4105_large_doc_small_upd 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt10897_compact_quick_interrupt/test_wt10897_compact_quick_interrupt 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt2695_checksum/test_wt2695_checksum 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt4156_metadata_salvage/test_wt4156_metadata_salvage 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt2447_join_main_table/test_wt2447_join_main_table 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt2592_join_schema/test_wt2592_join_schema 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt4117_checksum/test_wt4117_checksum 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt4699_json/test_wt4699_json 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt4891_meta_ckptlist_get_alloc/test_wt4891_meta_ckptlist_get_alloc 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt9937_parse_opts/test_wt9937_parse_opts 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt3363_checkpoint_op_races/test_wt3363_checkpoint_op_races 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt3120_filesys/test_wt3120_filesys -b $(pwd) 2>&1
+            ${test_env_vars|} $(pwd)/test/csuite/wt8659_reconstruct_database_from_logs/test_wt8659_reconstruct_database_from_logs 2>&1
+
+            # A subset of python tests that cover areas not touched by csuite
+            # In some cases use -s 1 to only run the first scenario for each test
+            # (To check. Some of these can probably be removed)
```

Comment:
Curious if it's an obsolete comment.

Reply:
That's correct - that comment was copy/pasted from the original competition entry, and is now obsolete. I'll remove it.